test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -416,6 +502,100 @@ describe('View recordset,', function() {
                 });
             });
 
+            it("apply sorting on different columns.", function () {
+                browser.get(browser.params.url + "/recordset/#" + browser.params.catalogId + "/product-recordset:" + accommodationParams.table_name + "?limit=3");
+                var EC = protractor.ExpectedConditions;
+                chaisePage.waitForElementInverse(element.all(by.id("spinner")).get(0));
+                browser.wait(EC.presenceOf(chaisePage.recordsetPage.getRows().get(2)), browser.params.defaultTimeout);
+                for (var j = 0; j < accommodationParams.sortedData.length; j++) {
+                    (function (k) {
+                        //Click on sort button
```

Comment:
Before clicking the sort button, let's verify the proper glyphicon is shown when no sorting is applied.